src/defaults.js

```diff
@@ -1,4 +1,4 @@
-import { TYPE_ELEMENT } from './constants';
+import binding from './polyfill/binding';
```

Comment:
I was thinking about that this morning actually. Probably better in something like `src/binding/w3c`. #230.